service/iot-service-client/src/main/java/com/microsoft/azure/sdk/iot/service/Device.java

```diff
@@ -69,94 +68,85 @@ public static Device createDevice(String deviceId, AuthenticationType authentica
+     * @param authenticationType The type of authentication used by this device.
      */
     private Device(String deviceId, AuthenticationType authenticationType)
     {
-        // Codes_SRS_SERVICE_SDK_JAVA_DEVICE_28_002: [The constructor shall set the deviceId and symmetricKey.]
         super(deviceId, authenticationType);
 
-        // Codes_SRS_SERVICE_SDK_JAVA_DEVICE_28_003: [The constructor shall initialize all properties to default values]
         this.setPropertiesToDefaultValues();
     }
 
-    // Codes_SRS_SERVICE_SDK_JAVA_DEVICE_12_001: [The Device class shall have the following properties: deviceId, Etag,
-    // SymmetricKey, ConnectionState, ConnectionStateUpdatedTime, LastActivityTime, symmetricKey, thumbprint, authentication]
-    /**
-     * "Enabled", "Disabled".
-     * If "Enabled", this device is authorized to connect.
-     * If "Disabled" this device cannot receive or send messages, and statusReason must be set.
-     */
     protected DeviceStatus status;
 
     /**
-     * Getter for DeviceStatus object
-     * @return The deviceStatus object
+     * Getter for DeviceStatus object.
+     * Values: {code "Enabled"}, {code "Disabled"}.
+     * If {@code "Enabled"}, this device is authorized to connect.
+     * If {@code "Disabled"} this device cannot receive or send messages, and {@link #statusReason} must be set.
+     *
+     * @return The deviceStatus object.
      */
     public DeviceStatus getStatus()
     {
         return status;
     }
 
     /**
-     * Setter for DeviceStatus object
+     * Setter for DeviceStatus object.
+     * Values: {code "Enabled"}, {code "Disabled"}.
+     * If {code "Enabled"}, this device is authorized to connect.
+     * If {code "Disabled"} this device cannot receive or send messages, and statusReason must be set.
      *
-     * @param status status to be set
+     * @param status status to be set.
      */
     public void setStatus(DeviceStatus status)
     {
         this.status = status;
     }
 
     /**
-     * A 128 char long string storing the reason of suspension.
-     * (all UTF-8 chars allowed).
+     * A 128 char long string storing the reason of suspension (all UTF-8 chars allowed).
      */
     protected String statusReason;
 
-    private String scope;
-
     /**
-     * Getter for status reason
+     * Getter for status reason.
      *
-     * @return The statusReason string
+     * @return The statusReason string.
      */
     public String getStatusReason()
     {
         return statusReason;
     }
 
-    /**
-     * Datetime of last time the state was updated.
-     */
     protected String statusUpdatedTime;
 
     /**
-     * Getter for status updated time string
+     * Getter for status updated on string.
      *
-     * @return The string containing the time when the statusUpdated parameter was updated
+     * @return The string containing the time when the {@link #statusUpdatedTime} object was updated.
```

Comment:
Do we really refer to the private fields? I find this part of Java weird. The field is the source of truth, but the getter and setter is how you access it. So what should the doc comments refer to?

Reply:
I'd be surprised if you could link to a non-public field in public javadocs. I'd revert back to what was here earlier

Reply:
Reverted. Should the text be updated to not refer to it? Seems weird to me to refer to private fields anyway.